schema/resolvers.js

```diff
@@ -197,9 +205,24 @@ const Resolvers = {
         return position;
       }
       return ctx.repositories.Section.getPosition({ id });
+    },
+    introduction: section => {
+      const { introductionEnabled } = section;
+
+      return introductionEnabled
```

Comment:
I think it's best to avoid ternaries over multiple lines.  Maybe this should be `if(...){...}else{...}` statement.

Reply:
Or (by using destructing and renaming) you could change this to:

`return introductionEnabled ? {title, content} : null;`

Here's a good article on destructuring: https://medium.com/@pyrolistical/destructuring-nested-objects-9dabdd01a3b8